### DISCUSSION TITLE
tab bar scrolling

### DISCUSSION THREAD

**Discussion by tksoh** [eteran/nedit-ng#283] [General]
@eteran 

When we have too many tabs the tab bar, NG will bring up the arrows on the RHS of the tab.  Right now it's a little clumsy when we hit the left end, we have to move the mouse all the way to the right end in order to click on the left arrow. Is it possible to move the left arrow to the left side of the bar, which should be more intuitive? 

![2021-01-17 11_40_52-Window](https://user-images.githubusercontent.com/494222/104830337-e68a9280-58b8-11eb-9d8f-ed2f09ae0fe2.jpg)

**Comment by tksoh** [comment]
BTW, having the left arrow on the left side should also provide clearer visual cue that there are more tabs hiding behind the left edge.

**Comment by eteran** [comment]
I'm not sure how much control we have over this, but I'll look into it.
